src/radio/radio-group.component.ts

```diff
@@ -229,7 +232,7 @@ export class RadioGroup implements AfterContentInit, AfterViewInit, ControlValue
 	updateSelectedRadioFromValue() {
 		let alreadySelected = this._selected != null && this._selected.value === this._value;
 		if (this.radios && !alreadySelected) {
-			if (this.selected) {
+			if (this.selected && this.value) {
```

Comment:
because there are instants in the lifecycle where `selected` has a value, but `value` hasn't and it was deselecting the `selected` without updating it from the `value` (eg. when one of the radio are provided with the property `[selected]="true"`